package.json

```diff
@@ -83,6 +83,7 @@
     "eslint-plugin-sonarjs": "^0.5.0",
```

Comment:
Note: I had to delete my build output folders (`.kompendium`, `dist`, `www`) to get this to work. Otherwise it got stuck on something, and after I canceled the build, the node_modules would be missing, so I had to run `npm install` again 😄

Reply:
Hmm ok :thinking: